ghproxy/ghproxy.go

```diff
@@ -155,6 +155,9 @@ func main() {
 	}
 
 	pjutil.ServePProf(o.instrumentationOptions.PProfPort)
+	if o.instrumentationOptions.ProfileMemory {
```

Comment:
Lets just move this and `pjutil.ServePprof` into a method of the instrumentation options (`Instrument`?) instead of repeating it everywhere?

Reply:
Yeah, we can do that too.